saleor/graphql/account/mutations/base.py

```diff
@@ -46,6 +49,48 @@ def can_edit_address(user, address):
     )
 
 
+def validate_channel(channel_slug):
```

Comment:
The identical function is used in the `CheckoutCreate` mutation (and `clean_channel` as well). Try to extract these functions and reuse them. They could be added to `graphql/channel/utils.py` maybe.